topico_laser: /rslidar_points
tamanho_fila_subscriber: 10
ganho_angular: 11
ganho_linear: 3
topico_amcl: /amcl_pose
topico_velocidade: /cmd_vel
topico_marcador: /visualization_marker
tamanho_objetivo: 1.0
alcance_controlador_local: 1.15
velocidade_angular_maxima: 0.9
topico_caminho_a_estrela: /Checkpoints

